src/main/java/uk/gov/justice/digital/hmpps/keyworker/config/RestTemplateConfiguration.java

```diff
@@ -57,31 +54,23 @@ public RestTemplate elite2ApiHealthRestTemplate(final RestTemplateBuilder restTe
     private RestTemplate getRestTemplate(final RestTemplateBuilder restTemplateBuilder, final String uri) {
         return restTemplateBuilder
                 .rootUri(uri)
-                .additionalInterceptors(getRequestInterceptors())
+                .additionalInterceptors(new JwtAuthInterceptor())
                 .build();
     }
 
     private RestTemplate getHealthRestTemplate(final RestTemplateBuilder restTemplateBuilder, final String uri) {
         return restTemplateBuilder
                 .rootUri(uri)
-                .additionalInterceptors(getRequestInterceptors())
+                .additionalInterceptors(new JwtAuthInterceptor())
```

Comment:
The request interceptor is replaced, what does this change mean?

Reply:
the method takes either a list or varargs, only need the JWT interceptor here as the W3C tracing one happens automatically with the java agent

Reply:
cool